.tekton/pull-request.yaml

```diff
@@ -154,11 +154,11 @@ spec:
         taskSpec:
           steps:
             - name: fail-when-repo-is-missed
-              image: registry.redhat.io/openshift4/ose-cli:v4.11
+              image: registry.access.redhat.com/ubi9/toolbox
```

Comment:
.tekton/scripts/check-task-pipeline-bundle-repos.sh: line 30: oc: command not found